README.md

```diff
@@ -72,7 +72,7 @@ System properties are preferred over environment variables. The following system
 
 | Property / Environment Variable                                                                                 | Description                                                                                                                              | Default value                                         |
 |-----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------|
-| `kubernetes.disable.autoConfig` / `KUBERNETES_DISABLE_AUTOCONFIG`                                               | Disable automatic configuration                                                                                                          | `false`                                               |
+| `kubernetes.disable.autoConfig` / `KUBERNETES_DISABLE_AUTOCONFIG`                                               | Disable automatic configuration (cluster, networking and authentication configuration, including all other entries in this table)        | `false`                                               |
```

Comment:
Umm, I think we need to reword it a little. I think this property turns off KubernetesClient's behavior to look for cluster-related configuration in well-known sources.